### ISSUE TITLE
Training results

### ISSUE THREAD

**Issue by Namzakku** [bakhuiyong/Camera-Motion-Detection-Using-CNN-in-Movie#1]
Hi! I'm curious what train, val loss and accuracy you got?
And it would be really nice if you can talk a little bit more about the dataset structure!

**Comment by ichlubna** [comment]
I tried to run their code and made it work. For anyone interested, this is what I did:

Download dataset with the link in the paper.
Use ALZip software to extract the egg archive.
Have train, validation and test folders at the scripts folder with the data.
Convert the notebooks to pypthon:
```
jupyter nbconvert CNN\ Based\ Detector.ipynb --to python
jupyter nbconvert Improved\ Extractor\ of\ Camera\ Motion.ipynb --to python
```
Create output folders
- dataset/11_frames/test,train,validation
- and have all categories folders in those three (pan, pedestal, tilt, truck, zoom_in, zoom_out)

Here I have edited the scripts:
- replaced the "train" folder hardcoded with sys.argv[1] in the extractor script
- had to import from tensorflow.keras.applications.resnet50 import ResNet50 in the detector script
- had to set the path to the extracted folders in the detector script:
```
    TRAIN_DIR = './dataset/11_frames/train'
    VALID_DIR = './dataset/11_frames/validation'
    caltech_dir = "dataset/11_frames/test/pan"
```
Extract the motion vectors from the dataset:
```
python Improved\ Extractor\ of\ Camera\ Motion.py train
python Improved\ Extractor\ of\ Camera\ Motion.py validation
python Improved\ Extractor\ of\ Camera\ Motion.py train
```
Use the extracted data in the training:
`python CNN\ Based\ Detector.py`
